src/gui/reports/ReportsWidgetPasskeys.cpp

```diff
@@ -289,6 +290,17 @@ void ReportsWidgetPasskeys::importPasskey()
 
 void ReportsWidgetPasskeys::exportPasskey()
 {
+    auto answer = MessageBox::question(
+        this,
+        tr("Export Confirmation"),
+        tr("You are about to export passkeys to unencrypted files. This will leave your "
+           "passkey and other sensitive information vulnerable! Are you sure you want to continue?"),
```

Comment:
No exclamation point, please. Perhaps just leave it as "The passkey file will be vulnerable to theft and unauthorized use, if left unsecured."

Reply:
FYI: Copied from ExportDialog.cpp :)

Reply:
Dang, your choice 🫠